### DISCUSSION TITLE
Expo + Fastlane + Screenshot

### DISCUSSION THREAD

**Discussion by dezerb** [expo/expo#20657] [Archived: Questions]
Hi folks,
I'm working with an expo project, but I wanted to automatise store upload process. I have a problem to find a valid solution how to use fastlane with expo in general to take and uploads the screenshots. 
I don't even know how to start should I use custom Gymfile in ios folder or prepare whole new fastlan config and use expo for for building package file?

**Comment by inakiarroyo** [comment]
Any update about this?
